guides/model-state.md

```diff
@@ -0,0 +1,13 @@
+## Avoid Local/UI state on Models
+
+Models live on the `store` service, and all services hold "application state."
+UI or local state should not be entangled with application state by adding
+it to a model.
```

Comment:
@mike-north I think the intended sentiment is to not set UI (non-persisted) state on models in particular.

ember-state-services is fine; it's doing things like setting non-persisted flags on `DS.Model` instances to drive UI state, or mutating a `hasMany` solely as a way of restricting what elements are rendered &c.

1. Do you agree with that sentiment?
2. Seems like the above could use a bit of wordsmithing to better convey the intent?